depends/patches/qt/fix_riscv_atomic.patch

```diff
@@ -0,0 +1,17 @@
+A workaround for error when building for riscv64.
+
+Upstream report: https://bugreports.qt.io/browse/QTBUG-84580
+
+Patch concept: https://bugs.gentoo.org/790689
+
+
+--- old/qtdeclarative/src/qml/qml.pro
++++ new/qtdeclarative/src/qml/qml.pro
+@@ -19,6 +19,7 @@ solaris-cc*:QMAKE_CXXFLAGS_RELEASE -= -O2
+ 
+ # Ensure this gcc optimization is switched off for mips platforms to avoid trouble with JIT.
+ gcc:isEqual(QT_ARCH, "mips"): QMAKE_CXXFLAGS += -fno-reorder-blocks
++gcc:isEqual(QT_ARCH, "riscv64"): LIBS += -latomic
```

Comment:
This differs form the mentioned patch by excluding the check for riscv32. This is ok as we do not release riscv32 binaries:

original patch's relevant line
```diff
+ gcc:if(isEqual(QT_ARCH, "riscv32")|isEqual(QT_ARCH, "riscv64")): LIBS += -latomic
```